2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Configure stats pid to 47198
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/.config/wandb/settings
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_resume_never_failure/wandb/settings
2022-12-19 20:33:06,364 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_REQUIRE_REPORT_EDITING_V0
2022-12-19 20:33:06,364 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_ERROR_REPORTING
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'username': 'test_resume_never_failure', 'base_url': 'http://localhost:65194', 'api_key': '***REDACTED***'}
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_init.py:_log_setup():476] Logging user logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_resume_never_failure/wandb/run-20221219_203306-4yhlzxck/logs/debug.log
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_resume_never_failure/wandb/run-20221219_203306-4yhlzxck/logs/debug-internal.log
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_init.py:init():516] calling init triggers
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-12-19 20:33:06,364 INFO    MainThread:47198 [wandb_init.py:init():569] starting backend
2022-12-19 20:33:06,365 INFO    MainThread:47198 [wandb_init.py:init():573] setting up manager
2022-12-19 20:33:06,372 INFO    MainThread:47198 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-12-19 20:33:06,372 INFO    MainThread:47198 [wandb_init.py:init():580] backend started and connected
2022-12-19 20:33:06,379 INFO    MainThread:47198 [wandb_init.py:init():658] updated telemetry
2022-12-19 20:33:06,410 INFO    MainThread:47198 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-19 20:33:06,540 ERROR   MainThread:47198 [wandb_init.py:init():713] encountered error: resume='never' but run (4yhlzxck) exists
2022-12-19 20:33:07,583 WARNING MsgRouterThr:47198 [router.py:message_loop():77] message_loop has been closed
2022-12-19 20:33:10,295 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Configure stats pid to 47198
2022-12-19 20:33:10,295 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/.config/wandb/settings
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_log_code_settings/wandb/settings
2022-12-19 20:33:10,296 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_REQUIRE_REPORT_EDITING_V0
2022-12-19 20:33:10,296 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_ERROR_REPORTING
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'username': 'test_log_code_settings', 'base_url': 'http://localhost:65194', 'api_key': '***REDACTED***'}
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_init.py:_log_setup():476] Logging user logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_log_code_settings/wandb/run-20221219_203310-1c98h671/logs/debug.log
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_log_code_settings/wandb/run-20221219_203310-1c98h671/logs/debug-internal.log
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_init.py:init():516] calling init triggers
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_init.py:init():569] starting backend
2022-12-19 20:33:10,296 INFO    MainThread:47198 [wandb_init.py:init():573] setting up manager
2022-12-19 20:33:10,303 INFO    MainThread:47198 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-12-19 20:33:10,307 INFO    MainThread:47198 [wandb_init.py:init():580] backend started and connected
2022-12-19 20:33:10,316 INFO    MainThread:47198 [wandb_init.py:init():658] updated telemetry
2022-12-19 20:33:10,404 INFO    MainThread:47198 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-19 20:33:10,594 INFO    MainThread:47198 [wandb_run.py:_on_init():2000] communicating current version
2022-12-19 20:33:10,886 INFO    MainThread:47198 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.13.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"
delete_message: "wandb version 0.13.5.dev1 has been retired!  Please upgrade."

2022-12-19 20:33:10,886 INFO    MainThread:47198 [wandb_init.py:init():728] starting run threads in backend
2022-12-19 20:33:11,304 INFO    MainThread:47198 [wandb_run.py:_console_start():1980] atexit reg
2022-12-19 20:33:11,304 INFO    MainThread:47198 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.OFF
2022-12-19 20:33:11,305 INFO    MainThread:47198 [wandb_init.py:init():765] run started, returning control to user process
2022-12-19 20:33:11,305 INFO    MainThread:47198 [wandb_run.py:_finish():1746] finishing run mock_server_entity/test/1c98h671
2022-12-19 20:33:11,306 INFO    MainThread:47198 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-12-19 20:33:11,306 INFO    MainThread:47198 [wandb_run.py:_restore():1932] restore
2022-12-19 20:33:11,306 INFO    MainThread:47198 [wandb_run.py:_restore():1938] restore done
2022-12-19 20:33:15,882 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-12-19 20:33:15,882 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-12-19 20:33:15,884 INFO    MainThread:47198 [wandb_run.py:_footer_sync_info():3333] logging synced files
2022-12-19 20:33:18,659 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Configure stats pid to 47198
2022-12-19 20:33:18,659 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/.config/wandb/settings
2022-12-19 20:33:18,659 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_local_warning__True-True-True__/wandb/settings
2022-12-19 20:33:18,660 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_REQUIRE_REPORT_EDITING_V0
2022-12-19 20:33:18,660 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_ERROR_REPORTING
2022-12-19 20:33:18,660 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'username': 'test_local_warning%5BTrue-True-True%5D', 'base_url': 'http://localhost:65194', 'api_key': '***REDACTED***'}
2022-12-19 20:33:18,660 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-19 20:33:18,661 INFO    MainThread:47198 [wandb_init.py:_log_setup():476] Logging user logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_local_warning__True-True-True__/wandb/run-20221219_203318-2kfhvg9h/logs/debug.log
2022-12-19 20:33:18,661 INFO    MainThread:47198 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_local_warning__True-True-True__/wandb/run-20221219_203318-2kfhvg9h/logs/debug-internal.log
2022-12-19 20:33:18,661 INFO    MainThread:47198 [wandb_init.py:init():516] calling init triggers
2022-12-19 20:33:18,664 INFO    MainThread:47198 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-12-19 20:33:18,667 INFO    MainThread:47198 [wandb_init.py:init():569] starting backend
2022-12-19 20:33:18,667 INFO    MainThread:47198 [wandb_init.py:init():573] setting up manager
2022-12-19 20:33:18,680 INFO    MainThread:47198 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-12-19 20:33:18,683 INFO    MainThread:47198 [wandb_init.py:init():580] backend started and connected
2022-12-19 20:33:18,692 INFO    MainThread:47198 [wandb_init.py:init():658] updated telemetry
2022-12-19 20:33:18,773 INFO    MainThread:47198 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-19 20:33:19,006 INFO    MainThread:47198 [wandb_run.py:_on_init():2000] communicating current version
2022-12-19 20:33:19,204 INFO    MainThread:47198 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.13.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"
delete_message: "wandb version 0.13.5.dev1 has been retired!  Please upgrade."

2022-12-19 20:33:19,204 INFO    MainThread:47198 [wandb_init.py:init():728] starting run threads in backend
2022-12-19 20:33:19,439 INFO    MainThread:47198 [wandb_run.py:_console_start():1980] atexit reg
2022-12-19 20:33:19,440 INFO    MainThread:47198 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.OFF
2022-12-19 20:33:19,440 INFO    MainThread:47198 [wandb_init.py:init():765] run started, returning control to user process
2022-12-19 20:33:19,441 INFO    MainThread:47198 [wandb_run.py:_finish():1746] finishing run mock_server_entity/test/2kfhvg9h
2022-12-19 20:33:19,441 INFO    MainThread:47198 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-12-19 20:33:19,441 INFO    MainThread:47198 [wandb_run.py:_restore():1932] restore
2022-12-19 20:33:19,441 INFO    MainThread:47198 [wandb_run.py:_restore():1938] restore done
2022-12-19 20:33:25,304 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-12-19 20:33:25,304 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-12-19 20:33:25,306 INFO    MainThread:47198 [wandb_run.py:_footer_sync_info():3333] logging synced files
2022-12-19 20:33:30,062 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Configure stats pid to 47198
2022-12-19 20:33:30,067 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/.config/wandb/settings
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_local_warning__False-False-False__/wandb/settings
2022-12-19 20:33:30,068 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_REQUIRE_REPORT_EDITING_V0
2022-12-19 20:33:30,068 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_ERROR_REPORTING
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'username': 'test_local_warning%5BFalse-False-False%5D', 'base_url': 'http://localhost:65194', 'api_key': '***REDACTED***'}
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_init.py:_log_setup():476] Logging user logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_local_warning__False-False-False__/wandb/run-20221219_203330-1swxhks7/logs/debug.log
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_local_warning__False-False-False__/wandb/run-20221219_203330-1swxhks7/logs/debug-internal.log
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_init.py:init():516] calling init triggers
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_init.py:init():569] starting backend
2022-12-19 20:33:30,068 INFO    MainThread:47198 [wandb_init.py:init():573] setting up manager
2022-12-19 20:33:30,078 INFO    MainThread:47198 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-12-19 20:33:30,085 INFO    MainThread:47198 [wandb_init.py:init():580] backend started and connected
2022-12-19 20:33:30,092 INFO    MainThread:47198 [wandb_init.py:init():658] updated telemetry
2022-12-19 20:33:30,149 INFO    MainThread:47198 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-19 20:33:30,352 INFO    MainThread:47198 [wandb_run.py:_on_init():2000] communicating current version
2022-12-19 20:33:30,650 INFO    MainThread:47198 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.13.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"
delete_message: "wandb version 0.13.5.dev1 has been retired!  Please upgrade."

2022-12-19 20:33:30,651 INFO    MainThread:47198 [wandb_init.py:init():728] starting run threads in backend
2022-12-19 20:33:31,090 INFO    MainThread:47198 [wandb_run.py:_console_start():1980] atexit reg
2022-12-19 20:33:31,090 INFO    MainThread:47198 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.OFF
2022-12-19 20:33:31,091 INFO    MainThread:47198 [wandb_init.py:init():765] run started, returning control to user process
2022-12-19 20:33:31,092 INFO    MainThread:47198 [wandb_run.py:_finish():1746] finishing run mock_server_entity/test/1swxhks7
2022-12-19 20:33:31,092 INFO    MainThread:47198 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-12-19 20:33:31,093 INFO    MainThread:47198 [wandb_run.py:_restore():1932] restore
2022-12-19 20:33:31,093 INFO    MainThread:47198 [wandb_run.py:_restore():1938] restore done
2022-12-19 20:33:37,632 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-12-19 20:33:37,632 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-12-19 20:33:37,634 INFO    MainThread:47198 [wandb_run.py:_footer_sync_info():3333] logging synced files
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Configure stats pid to 47198
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/.config/wandb/settings
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_artifact_string_digest_run_config_set_item/wandb/settings
2022-12-19 20:33:40,082 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_REQUIRE_REPORT_EDITING_V0
2022-12-19 20:33:40,082 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_ERROR_REPORTING
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'username': 'test_artifact_string_digest_run_config_set_item', 'base_url': 'http://localhost:65194', 'api_key': '***REDACTED***'}
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_init.py:_log_setup():476] Logging user logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_artifact_string_digest_run_config_set_item/wandb/run-20221219_203340-uuzi1hdf/logs/debug.log
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_artifact_string_digest_run_config_set_item/wandb/run-20221219_203340-uuzi1hdf/logs/debug-internal.log
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_init.py:init():516] calling init triggers
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_init.py:init():569] starting backend
2022-12-19 20:33:40,082 INFO    MainThread:47198 [wandb_init.py:init():573] setting up manager
2022-12-19 20:33:40,089 INFO    MainThread:47198 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-12-19 20:33:40,090 INFO    MainThread:47198 [wandb_init.py:init():580] backend started and connected
2022-12-19 20:33:40,099 INFO    MainThread:47198 [wandb_init.py:init():658] updated telemetry
2022-12-19 20:33:40,144 INFO    MainThread:47198 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-19 20:33:40,307 INFO    MainThread:47198 [wandb_run.py:_on_init():2000] communicating current version
2022-12-19 20:33:40,605 INFO    MainThread:47198 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.13.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"
delete_message: "wandb version 0.13.5.dev1 has been retired!  Please upgrade."

2022-12-19 20:33:40,605 INFO    MainThread:47198 [wandb_init.py:init():728] starting run threads in backend
2022-12-19 20:33:40,858 INFO    MainThread:47198 [wandb_run.py:_console_start():1980] atexit reg
2022-12-19 20:33:40,858 INFO    MainThread:47198 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.OFF
2022-12-19 20:33:40,861 INFO    MainThread:47198 [wandb_init.py:init():765] run started, returning control to user process
2022-12-19 20:33:40,886 INFO    MainThread:47198 [wandb_config.py:__setitem__():148] config set dataset = <Artifact abc123> - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x2c6195430>>
2022-12-19 20:33:40,886 INFO    MainThread:47198 [wandb_run.py:_config_callback():1160] config_cb dataset <Artifact abc123> None
2022-12-19 20:33:40,887 INFO    MainThread:47198 [wandb_run.py:_finish():1746] finishing run mock_server_entity/test/uuzi1hdf
2022-12-19 20:33:40,887 INFO    MainThread:47198 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-12-19 20:33:40,887 INFO    MainThread:47198 [wandb_run.py:_restore():1932] restore
2022-12-19 20:33:40,887 INFO    MainThread:47198 [wandb_run.py:_restore():1938] restore done
2022-12-19 20:33:44,589 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-12-19 20:33:44,590 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-12-19 20:33:44,592 INFO    MainThread:47198 [wandb_run.py:_footer_sync_info():3333] logging synced files
2022-12-19 20:33:48,039 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Configure stats pid to 47198
2022-12-19 20:33:48,039 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /Users/estellaxin/.config/wandb/settings
2022-12-19 20:33:48,039 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from /private/var/folders/vh/k98fyx251l792bbzj485tdnw0000gp/T/tmpfoxcajmf/wandb/settings
2022-12-19 20:33:48,039 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_REQUIRE_REPORT_EDITING_V0
2022-12-19 20:33:48,039 WARNING MainThread:47198 [wandb_setup.py:_flush():68] Unknown environment variable: WANDB_ERROR_REPORTING
2022-12-19 20:33:48,039 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'username': 'test_wandb_artifact_init_config', 'base_url': 'http://localhost:65194', 'api_key': '***REDACTED***'}
2022-12-19 20:33:48,039 INFO    MainThread:47198 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-12-19 20:33:48,039 INFO    MainThread:47198 [wandb_init.py:_log_setup():476] Logging user logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_wandb_artifact_init_config/wandb/run-20221219_203348-jffjz0kc/logs/debug.log
2022-12-19 20:33:48,040 INFO    MainThread:47198 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/estellaxin/Projects/wandb/tests/unit_tests_old/logs/test_wandb_artifact_init_config/wandb/run-20221219_203348-jffjz0kc/logs/debug-internal.log
2022-12-19 20:33:48,040 INFO    MainThread:47198 [wandb_init.py:init():516] calling init triggers
2022-12-19 20:33:48,040 INFO    MainThread:47198 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-12-19 20:33:48,040 INFO    MainThread:47198 [wandb_init.py:init():569] starting backend
2022-12-19 20:33:48,040 INFO    MainThread:47198 [wandb_init.py:init():573] setting up manager
2022-12-19 20:33:48,047 INFO    MainThread:47198 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-12-19 20:33:48,048 INFO    MainThread:47198 [wandb_init.py:init():580] backend started and connected
2022-12-19 20:33:48,055 INFO    MainThread:47198 [wandb_init.py:init():658] updated telemetry
2022-12-19 20:33:48,056 INFO    MainThread:47198 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-12-19 20:33:48,225 INFO    MainThread:47198 [wandb_run.py:_on_init():2000] communicating current version
2022-12-19 20:33:48,546 INFO    MainThread:47198 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.13.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"
delete_message: "wandb version 0.13.5.dev1 has been retired!  Please upgrade."

2022-12-19 20:33:48,546 INFO    MainThread:47198 [wandb_init.py:init():728] starting run threads in backend
2022-12-19 20:33:48,836 INFO    MainThread:47198 [wandb_run.py:_config_callback():1160] config_cb None None {'test_reference_download': <wandb.sdk.wandb_artifacts.Artifact object at 0x2c61d2190>}
2022-12-19 20:33:48,839 INFO    MainThread:47198 [wandb_run.py:_console_start():1980] atexit reg
2022-12-19 20:33:48,840 INFO    MainThread:47198 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.OFF
2022-12-19 20:33:48,840 INFO    MainThread:47198 [wandb_init.py:init():765] run started, returning control to user process
2022-12-19 20:33:48,840 INFO    MainThread:47198 [wandb_run.py:_finish():1746] finishing run mock_server_entity/test/jffjz0kc
2022-12-19 20:33:48,841 INFO    MainThread:47198 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-12-19 20:33:48,841 INFO    MainThread:47198 [wandb_run.py:_restore():1932] restore
2022-12-19 20:33:48,841 INFO    MainThread:47198 [wandb_run.py:_restore():1938] restore done
2022-12-19 20:33:52,538 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-12-19 20:33:52,538 INFO    MainThread:47198 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-12-19 20:33:52,539 INFO    MainThread:47198 [wandb_run.py:_footer_sync_info():3333] logging synced files
